src/plots/gl2d/scene2d.js

```diff
@@ -359,7 +358,6 @@ proto.destroy = function() {
 
     if(!this.staticPlot) this.container.removeChild(this.canvas);
     this.container.removeChild(this.svgContainer);
-    this.container.removeChild(this.mouseContainer);
```

Comment:
So wait, are we :hocho: the gl2d picking routine?

It would be nice to merge the cartesian and gl2d hover logic at some point, but swapping in cartesian hover in gl2d probably means poor performance on graphs with 1e5 and up data points at this stage.